geonode/upload/upload.py

```diff
@@ -891,6 +891,7 @@ def final_step(upload_session, user, charset="UTF-8", layer_id=None):
                             name=_vals.get('name'),
                             abstract=_vals.get('abstract', _('No abstract provided')),
                             owner=user,
+                            dirty_state=True,
```

Comment:
Why do we create the layer in a dirty state here? This was not available in the master PR. Is this wanted?

Reply:
Removed